platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -231,14 +248,23 @@ private MigrationReport doImport(
                 + productVersion
                 + ConfigurationMigrationManager.EXPORT_EXTENSION);
 
+    final Path keyFile = exportDirectory.resolve(ConfigurationMigrationManager.EXPORT_KEY_NAME);
+
     try {
+      Validate.isTrue(checksumIsValid(exportFile));
```

Comment:
❓ Should we really let an invalid checksum be reported as an internal error below? I think a specific error messages is warranted here.